### ISSUE TITLE
Running extensions in both Chrome and Firefox leaves temporary directories

### ISSUE THREAD

**Issue by ghostwords** [mozilla/web-ext#1981]
### Is this a feature request or a bug?
Bug

### What is the current behavior?
Running extensions in both Chrome and Firefox still leaves temporary directories. This happens despite `web-ext` logging that it cleans up some temp directory on exit in verbose mode. (The directory logged is indeed missing, but others remain.)

Here are some recent leftover folders in `/tmp/`:
```
.com.google.Chrome.tV0jJO
tmp-web-ext--2445-Rid4u33i0TaN
.com.google.Chrome.1YiHq1
tmp-web-ext--9065-2sTtAd1ZXCx2
lighthouse.cc52DU2
lighthouse.elq0cnz
lighthouse.1p1kCZO
lighthouse.aEuUs7s
lighthouse.GxOVGAS
lighthouse.LTtzBYw
vxRzd7e
lighthouse.mXLE5jI
tmp-web-ext--8248-8w2G2D5WDdNB
```

The "lighthouse", "Chrome" and "vxRzd7e"-like folders come from Chrome, I think. "tmp-web-ext" folders are from Firefox.

"lighthouse" folders in particular seem to sometimes grow to hundreds of megabytes in size when a `web-ext run` session is left running for days.

### Versions
web-ext 5.0.0
Various versions (both latest and older) of Firefox and Chrome.

**Comment by ghostwords** [comment]
#1957 sounds related.

**Comment by Rob--W** [comment]
Confirmed with `-t chromium`, but not reproducible with Firefox. Are you seeing this with `web-ext run` (Firefox)?

We should either ensure that the directory content is destroyed, or call `tmp.setGracefulCleanup()` in https://github.com/mozilla/web-ext/blob/cfc49ce803a9a320d06af5871a2e3aa8a8c533b5/src/util/temp-dir.js

**Comment by ghostwords** [comment]
Yes, the tmp-web-ext folders are from Firefox runs. I just tried again with the latest version of Firefox and got two new folders looking with names like "Temp-075a8052-2cd9-4460-8ac9-13e76c3aa77e". Plus there is a bunch of "rust_mozprofile.pspDgJWpvtWe" folders too, probably from Firefox. Again, these are all in `/tmp`. It's possible some of these are not from `web-ext`, but I'm sure a lot of them are.

**Comment by darkred** [comment]
I'd like to add the following I've noticed:

I have web-ext 6.0.0 , 
and Chrome stable 89 and Canary 91 (both x64) on win10 (Chrome stable is my default browser)



I use `npm run start` which internally launches Canary (via `web-ext run --target=chromium`).

Then, while Canary runs, a `tmp-web-ext` folder is indeed created in `%temp%`.
Additionally, the Canary profile is located in 
`C:\Users\user\AppData\Local\Google\Chrome SxS\Application\test\web-ext-profile\Default`

If I press Ctrl+C in the web-ext in console, or just close Canary window, and web-ext disconnects,
then the `tmp-web-ext` folder in `%temp%` is indeed removed,.
but the `C:\Users\user\AppData\Local\Google\Chrome SxS\Application\test` and its contents remains,
and if I re-launch `npm run start` it re-uses the profile from before.

**Is this the expected behavior? 
Shouldn't the `C:\Users\user\AppData\Local\Google\Chrome SxS\Application\test` folder also be removed on disconnect?**
